cli/cmd/generate_test.go

```diff
@@ -16,7 +16,7 @@ spec:
   name: "test"
 
   # Version of the plugin to use.
-  version: "development"
+  version: "v1.1.4"
```

Comment:
The tests shows the bug actually.
I'll update this test in https://github.com/cloudquery/cloudquery/pull/1848 to avoid having a hard coded version as that will break when we release the test plugin